src/encoded/static/components/body_map.js

```diff
@@ -126,6 +126,34 @@ const addingClass = (changedClass, matchingString, removeFlag = false) => {
     }
 };
 
+// Set body map diagram colors based on url, when component mounts
```

Comment:
I recommend using JSDocs rather than a vanilla comment as JSDocs are used by IDEs to provide information to consumers this function.

Reply:
I should definitely think about using JSDocs. There's one thing I don't like about the syntax - it can be difficult to distinguish between the variable name and the definition/explanation of the variable - but I think there are a number of other benefits - the enforcement of defining input parameters and returned values seems very useful.